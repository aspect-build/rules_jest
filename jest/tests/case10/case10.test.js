test("the config + setup files run", () => {
  expect(TEST_SETUP_VALUE).toEqual(42);
  expect(TEST_SETUP_VALUE).toMatchSnapshot();

  expect(TEST_CONFIG_VALUE).toBe(true);
  expect(TEST_CONFIG_VALUE).toMatchSnapshot();

  expect(TEST_CONFIG_INIT_VALUE).toBe(true);
  expect(TEST_CONFIG_INIT_VALUE).toMatchSnapshot();
});
