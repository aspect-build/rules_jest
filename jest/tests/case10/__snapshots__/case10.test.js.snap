// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the config + setup files run 1`] = `42`;

exports[`the config + setup files run 2`] = `true`;

exports[`the config + setup files run 3`] = `true`;
